pkg/templates/templates/infinispan-base-14.xml

```diff
@@ -37,13 +37,11 @@
         <socket-binding name="default" port="${infinispan.bind.port:11222}"/>
     </socket-bindings>
     <security>
-        {{ if or .Keystore.Password .Truststore.Path }}
```

Comment:
So it's always enabled even if user does not define any credential store?

Reply:
I've updated this so that we now create two credential-stores as discussed on Zulip. The user credential-store will only be created if the user has provided one or more credentials.